grammar.y

```diff
@@ -1129,10 +1131,16 @@ table_constraint_list_opt:
 table_constraint_list:
   ',' table_constraint
   {
+    if _, ok := $2.(*TableConstraintPrimaryKey); ok {
```

Comment:
If we see a `TableConstraintPrimaryKey` we set the flag.

Reply:
Should we check also the case where the user is creating the primary key in the same column definition?

Maybe explained better with an example that should fail:
`CREATE TABLE foo (bar INTEGER PRIMARY KEY, zar INTEGER, PRIMARY KEY(zar)`

Feels to me `table_constraint_list` is only setting the flag for the case where the primary key is defined at the table level, but not column level. Maybe that is the case?

Reply:
Oh yeah. missed that, thanks! I'll do it